janusgraph-es/src/main/java/org/janusgraph/diskstorage/es/ElasticSearchIndex.java

```diff
@@ -323,6 +328,7 @@
     private final boolean useAllField;
     private final Map<String, Object> ingestPipelines;
     private final boolean useMappingForES7;
+    private final long clientKeepAlive;
```

Comment:
Remove unused parameter. I don't see this parameter to be used anywhere except assigning a value to it

Reply:
Yes, good catch.